muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/records.tsx

```diff
@@ -260,7 +260,7 @@ class Records extends React.Component<RecordsProps, RecordsState> {
     {studentRecords}    
     
     <div className="application-sub-panel">
-      <div className="application-sub-panel__header text text--studies-header">{this.props.i18n.text.get("plugin.records.files.title")}</div>
+      <div className="application-sub-panel__header">{this.props.i18n.text.get("plugin.records.files.title")}</div>
```

Comment:
Why not use  an className as application-sub-panel__header-title as used in application-panel__header-title element to keep consistency.

Reply:
Look above the above